great_expectations/experimental/datasources/sql_datasource.py

```diff
@@ -306,9 +306,6 @@ class _SQLAsset(DataAsset):
     column_splitter: Optional[ColumnSplitter] = None
     name: str
 
-    # Internal attributes
```

Comment:
This extra declaration is not needed -- top-level `_DataAsset` already declares a `_datasource` private variable (and we are not counting on any special methods that might be found in the `SQLDatasource` extension).  Hence, deleting this is acceptable (and tests pass to confirm this).